microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -693,6 +695,49 @@ def verify_dpdk_send_receive_gb_hugepages_netvsc(
         except UnsupportedPackageVersionException as err:
             raise SkippedException(err)
 
+    @TestCaseMetadata(
+        description=(
+            """
+                Run the L3 forwarding test for DPDK
```

Comment:
Describe more details, on the test logic and test purpose.

Reply:
fixed